src/types/FilterType.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilterType {
```

Comment:
you can create enum for errors as well.

Reply:
Enum was in ErrorNotification component, but I placed it in separate file)